site/app/controllers/OfficeHoursQueueController.php

```diff
@@ -396,13 +396,10 @@ public function changeToken($queue_code) {
 
     /**
     * @Route("/{_semester}/{_course}/office_hours_queue/current_queue", methods={"GET"})
-    * @return MultiResponse
     */
     public function showCurrentQueue() {
         if (!$this->core->getConfig()->isQueueEnabled()) {
-            return MultiResponse::RedirectOnlyResponse(
-                new RedirectResponse($this->core->buildCourseUrl(['home']))
-            );
+            return;
```

Comment:
All routes must return a Response of some sort, or else you end up with a page with just a sidebar and a totally blank middle part.